apps/boundary/serializers.py

```diff
@@ -5,4 +5,4 @@
 class BoundarySerializer(KLPSerializer):
     class Meta:
         model = Boundary
-        fields = ('id', 'name', 'dise_slug', 'boundary_type', 'type', 'status')
+        fields = ('id', 'name', 'parent','dise_slug', 'boundary_type', 'type', 'status')
```

Comment:
Reiterating what I mentioned before - have a flake8 checker in place so that the code formatting stays consistent. For instance, there should be a space after the comma.